lib/services/route.dart

```diff
@@ -22,17 +20,18 @@ class HikingRoute {
     this.pointsOfInterest = pointsOfInterest;
     this.elevations = elevations;
     this.date = DateFormat('yyyy-MM-dd HH:mm').format(DateTime.now());
-    this.title = 'Sample title'; // TODO get region or custom info as title
+    setTitle('');
   }
 
   Future<String> buildTitle() async{
```

Comment:
this method should be removed from the route object and the title should just be fetched from latlng coords every time its needed. Otherwise we would have problems with sharing routes for example (we then have to share also the title).